phc-rails/app/models/account.rb

```diff
@@ -41,7 +41,80 @@ def self.spawn(params)
     if account.save then account else nil end
 
     #### TODO: POST TO SALESFORCE #####
+  end
+
+  def self.find_duplicates_by(attrs, count, cursor)
+    groups = {}
+    # Escape each attribute with quotes to prevent lowercasing by Rails
+    attrs.map! { |x| "\"#{x}\"" }
+    result = Account.select(attrs).group(attrs).having("count(*)>1")
+    result[cursor..cursor+count].each do |account|
+      fields = account.attributes
+      fields.delete('id')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.